Source/Core/Core/IOS/USB/Bluetooth/BTReal.h

```diff
@@ -65,10 +65,10 @@ class BluetoothReal final : public BluetoothBase
 
 private:
   static constexpr u8 INTERFACE = 0x00;
-  // Arbitrarily chosen value that allows emulated software to send commands often enough
-  // so that the sync button event is triggered at least every 200ms.
+  // Arbitrarily chosen value that allows emulated software to not block if libusb transfers
```

Comment:
AFAIK, Java_WiimoteAdapter.java is used for the DolphinBar (and only on Android). It doesn't need to be reverted, but I feel like allowing higher latency wouldn't be very useful on Android if usbip cannot be used in most cases.